virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -510,3 +510,30 @@ def check_in_vm(vm, target, old_parts, is_equal=False):
     except Exception as e:
         logging.error(str(e))
         return False
+
+
+def create_remote_disk_by_first(vm, params):
```

Comment:
If this function aims to create empty image on remote host, I assume it is not related to specific VM.
So vm parameter may not be used

Reply:
Unfortunately, I can not remove vm parameter because vm is used in the codes. See L528
` blk_source = get_first_disk_source(vm)`